src/github/entities/allowed-organization.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { OrganizationStatus } from 'src/common/constants';
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity()
+export class AllowedOrganization extends Base {
+  @Column({ type: 'varchar', length: 255, default: 'github' })
+  host: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true })
+  organizationName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminUserName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminEmail: string;
+
+  @Column('text', { array: true, default: [] })
+  activeUsers: string[];
+
+  @Column('text', { array: true, default: [] })
+  inactiveUsers: string[];
+
+  @Column({
+    type: 'enum',
+    enum: OrganizationStatus,
+    default: OrganizationStatus.ACTIVE,
+  })
+  status: OrganizationStatus;
+
+  @Column({ type: 'boolean', default: true })
+  allowAllUsers: boolean;
```

Comment:
PullProofAI

The `status` field uses an enum `OrganizationStatus`, which is a good practice for maintaining consistency. Make sure to handle enum values correctly throughout the application to avoid unexpected behavior.

```diff
+ // Ensure correct handling of enum values for OrganizationStatus in the status field
```

### Summary
Overall, the `AllowedOrganization` entity implementation looks solid. Adding validation logic, organizing columns, and ensuring proper array handling will further enhance its robustness and maintainability.



<!-- This is an auto-generated comment by PullProofAI -->